### PR TITLE
renewing sysbench arm support

### PR DESCRIPTION
**Pull request by ericavella**
the arm64/table count issue is fixed! so removing exception against setting num of tables in arm64
also expanding profile scenarios
and to top it all off adding a state property to track if the executables were initialized.

### CODE REVIEW

File: src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json [hunk 1]
```diff
@@ -27,12 +27,124 @@
...
                 "DatabaseScenario": "$.Parameters.DatabaseScenario",
                 "Role": "Client",
-                "Duration": "00:30:00",
+                "Duration": "02:00:00",
                 "RecordCount": 10000,
                 "Workload": "oltp_read_write",
                 "PackageName": "sysbench",
                 "SkipInitialize": "$.Parameters.SkipInitialize"
             }
+        },
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json:36]: Thank you for adding this. Does all these work with all-configurations? in-memory and mix etc?

File: src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs [hunk 2]
```diff
@@ -140,24 +140,37 @@ protected void InitializeApiClients()
...
-                await this.SystemManager.MakeFileExecutableAsync(
-                    this.Combine(scriptsDirectory, "in-memory.sh"),
-                    this.Platform,
-                    cancellationToken);
+                // install sysbench using repo scripts
+                if (this.Platform == PlatformID.Unix && this.DatabaseScenario != SysbenchOLTPScenario.Default)
+                {
+                    string scriptsDirectory = this.PlatformSpecifics.GetScriptPath("sysbencholtp");
+
+                    await this.SystemManager.MakeFileExecutableAsync(
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs:155]: There is one single method to make all files under one dir executable if you want

File: src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs [hunk 3]
```diff
@@ -267,6 +280,19 @@ public bool SysbenchInitialized
                 }
             }
 
+            public bool ExecutablesInitialized
```
Comment (yangpanMS) [inline src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs:283]: Let's not expose this in parameters. Saving in state is ok, but user shouldn't need to know it

File: src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json (hunk 1 above)
Comment (ericavella) [reply src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json:37]: yes it does * now * !

File: src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs (hunk 3 above)
Comment (ericavella) [reply src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs:273]: gotcha -- does that mean making this private?

Comment (yangpanMS) [reply src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs:273]: no I mean making it a property in the MySQL state, so user don't need to know whether they need to pass in this parameter

**Review by yangpanMS — APPROVED** [review]

**Pull request merged**